lib/src/resources/texture_atlas_format_json.dart

```diff
@@ -15,6 +15,9 @@ class _TextureAtlasFormatJson extends TextureAtlasFormat {
     final image = meta['image'] as String;
     final renderTextureQuad = await loader.getRenderTextureQuad(image);
 
+    //  Set texture info based on meta format
+    _setTextureFormat(renderTextureQuad.renderTexture, meta['format'] as String);
```

Comment:
Yes! The value `format` may be null – at least for pop-pop-win!

Reply:
Understood, will fix, most likely this evening...

Reply:
PR here: https://github.com/bp74/StageXL/pull/355